src/api/services/users.service.js

```diff
@@ -0,0 +1,26 @@
+const User = require("../../models/User");
+
+exports.signup = async userData => {
+  const { email, displayedName, profile } = userData;
+
+  if (!email || !displayedName || !profile) {
+    return new Error("이메일 또는 별명, 프로필 사진이 없습니다.");
+  }
+
+  try {
+    await User.create({
+      email,
+      nickName: displayedName,
+      profile,
+      createdGoals: [],
+      createdTodoIds: [],
+      token: null,
+    });
```

Comment:
이부분도 User스키마에서 displayName으로 바꿀예정입니다.

```suggestion
    await User.create({
      email,
      displayName,
      profile,
      createdGoals: [],
      createdTodoIds: [],
      token: null,
    });
```
으로 바꿔주시면 감사드립니다~